<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>800</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1200</width>
      <height>800</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1677226</width>
      <height>1677215</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>5</width>
      <height>0</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab{width:360}
QTabBar::tab{height:50}
/*设置TabWidget中QTabBar的样式*/
QTabBar::tab{
    background-color: #afcff9;
	font-family:Consolas;    /*设置tab中的文本的字体*/
	font-size:15pt;
border-bottom-left-radius: 10px;    
border-bottom-right-radius: 10px;
	color:#000000;    /*设置tab中的文本的颜色*/
	border-top-left-radius: 10px;    /*设置tab的边框的圆角（左上圆角）*/
	border-top-right-radius: 10px;    /*设置tab的边框的圆角（右上圆角）*/
	min-width: 25px;
	padding: 20px;
}
/*设置TabWidget中QTabBar的tab被选中时的样式*/
QTabBar::tab:selected{
    background-color: rgb(175, 214, 243, 65);
}
/*设置TabWidget中鼠标悬浮在QTabBar的tab上，但未选中该Tab的样式*/
QTabBar::tab:hover:!selected {
   
	background-color: rgb(225, 210, 255);
}
 
/*设置TabWidget的边框的样式*/
QTabWidget::pane {
    border: 1px solid rgb(175, 214, 243,65);
}
 
/*当打开多个tab，右侧出现，点击后，可以向前向后的按钮的样式*/
QTabBar QToolButton {
    border: none;
	color: rgb(255, 206, 6);
    background-color: #0b0e11;
}
 
QTabBar QToolButton:hover {
	background-color: #161a1e; 
}

</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>     day schedule    </string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>931</width>
        <height>701</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/*表格的一种美化方式*/
QHeaderView
{
    background:transparent;
}

QHeaderView::section
{
    font-size:14px;
    font-family:&quot;Microsoft YaHei&quot;;
    color:#FFFFFF;
    background:#60669B;
    border:none;
    text-align:left;
    min-height:49px;
    max-height:49px;
min-width:120px;
    margin-left:0px;
    padding-left:0px;

}

QTableWidget
{
    background:#FFFFFF;
    border:none;

    font-size:20px;
    font-family:&quot;Microsoft YaHei&quot;;
    color:#666666;
}
QTableWidget::item
{
    border-bottom:1px solid #EEF1F7 ;
}
QTableWidget::item::hover{
	background-color: #EFF4FF ;
}
QTableWidget::item::selected
{
    
	color: rgb(27, 27, 136);
    background:rgb(211, 221, 255);
}


QScrollBar::handle:vertical
{
    background: rgba(255,255,255,20%);
    border: 0px solid grey;
    border-radius:3px;
    width: 8px;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
{
    background:rgba(255,255,255,10%);
}


QScollBar::add-line:vertical, QScrollBar::sub-line:vertical
{
    background:transparent;
}
</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="themeEdit">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>160</y>
        <width>201</width>
        <height>421</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(211, 230, 255);
font: 18pt &quot;楷体&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>120</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Sitka Subheading&quot;;</string>
      </property>
      <property name="text">
       <string>theme</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>20</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QDateEdit
{
	font: 20pt &quot;Microsoft YaHei UI&quot;;
font-size: 20px;
background-color: rgb(212, 231, 255);
spacing: 5px;
border-top-left-radius: 5px;    /*设置tab的边框的圆角（左上圆角）*/
border-top-right-radius: 5px;    /*设置tab的边框的圆角（右上圆角）*/
border-bottom-left-radius: 5px; 
border-bottom-right-radius: 5px;
}

QDateEdit::drop-down {
width: 50px;                   //宽度（高度随控件）
border-left-width: 1px;   //加左边框线（可以不要，下同）
border-left-style: solid;  //左边框线类型
border-left-color: gray;  //左边框线颜色
}
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="themeConfirm">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>610</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-top-left-radius: 15px;    /*设置tab的边框的圆角（左上圆角）*/
border-top-right-radius: 15px;    /*设置tab的边框的圆角（右上圆角）*/
border-bottom-left-radius: 15px; 
border-bottom-right-radius: 15px;
font: 18pt &quot;Microsoft YaHei UI&quot;;
background-color: rgb(202, 223, 255);
}
QPushButton:pressed{
 background-color: rgb(175, 214, 243, 65);
}
QPushButton:hover{
background-color: rgb(225, 210, 255);
}</string>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>     month schedule     </string>
     </attribute>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>811</width>
        <height>651</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QCalendarWidget{
	font: 16pt &quot;黑体&quot;;
	alternate-background-color: rgb(0,0,0);

}
</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="dayscheduleEdit">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>110</y>
        <width>231</width>
        <height>461</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;楷体&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>40</y>
        <width>221</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>今日日程</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="save">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>620</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-top-left-radius: 15px;    /*设置tab的边框的圆角（左上圆角）*/
border-top-right-radius: 15px;    /*设置tab的边框的圆角（右上圆角）*/
border-bottom-left-radius: 15px; 
border-bottom-right-radius: 15px;
font: 18pt &quot;Microsoft YaHei UI&quot;;
background-color: rgb(202, 223, 255);
}
QPushButton:pressed{
 background-color: rgb(175, 214, 243, 65);
}
QPushButton:hover{
background-color: rgb(225, 210, 255);
}</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>     curriculum schedule     </string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1201</width>
        <height>721</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>30</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-top-left-radius: 15px;    /*设置tab的边框的圆角（左上圆角）*/
border-top-right-radius: 15px;    /*设置tab的边框的圆角（右上圆角）*/
border-bottom-left-radius: 15px; 
border-bottom-right-radius: 15px;
font: 30pt &quot;Microsoft YaHei UI&quot;;
background-color: rgb(202, 223, 255);
}
QPushButton:pressed{
 background-color: rgb(175, 214, 243, 65);
}
QPushButton:hover{
background-color: rgb(225, 210, 255);
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
